content-src/asrouter/templates/SimpleBelowSearchSnippet/_SimpleBelowSearchSnippet.scss

```diff
@@ -91,7 +91,12 @@
     }
 
     @media (max-width: 865px) {
-      margin: 0 60px 0 0;
+      margin: 0;
+      margin-inline-end: 60px;
+    }
+
+    @media (max-width: 609px) {
```

Comment:
Still better to have a reference variable than an seemingly arbitrary number. You should be able to do `$var - 1px` ? and a comment about the off-by-1 difference between min-width and max-width ?

Reply:
Or I suppose at least have a comment

Reply:
I'll set it that way with a comment - I don't know if it would be worth refactoring the breakpoints in future so that they overlap a little, to prevent those weird gaps?